python_modules/dagster/dagster_tests/core_tests/pythonic_config_tests/test_pythonic_config_types.py

```diff
@@ -775,6 +775,27 @@ def a_job():
         a_job.execute_in_process({"ops": {"a_struct_config_op": {"config": {"an_enum": "foo"}}}})
 
 
+def test_literal() -> None:
```

Comment:
Would be worth testing a literal field on a resource too, just because the codepaths that ends up hitting are slightly different